### PR TITLE
Create GIT.md

### PR DESCRIPTION
**Pull request by dreisen**

### CODE REVIEW

File: GIT.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+git fetch upstream
+git rebase upstream/master
+````
+4. Create a new branch to work on.  
+`git checkout -b my_feature`
+5. Commit changes on your feature_branch.
+6. Rebase your changes ontop of the upstream.  
+`git rebase upstream/master`
+7. Push your commits to your fork (`origin`).  
+`git push -f origin my_feature`
```
Comment (m-Schlitzer) [inline GIT.md]: We shouldn't force push at any time except in situations where it can't be fixed. This will only give us trouble.

File: GIT.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+8. Open a pull request on the original repository (`upstream`).
```
Comment (m-Schlitzer) [inline GIT.md]: In the pull request, link your issue or describe your feature. Explain why you changed something.

File: GIT.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# How we use git
+1. Fork the repository you want to work on.
+2. Set up two remotes.  
+`origin` - Which points to your fork (e.g. reisend/timetable-client)
```
Comment (m-Schlitzer) [inline GIT.md]: You don't need to set up origin, this is automatically added.

**Comment by m-Schlitzer** [comment]
Also, you might want to give the file a more fitting name, like USING_GIT.md, so it is instantly clear what is in the file.

**Comment by dreisen** [comment]
addressed comments

File: USING_GIT.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # How we use git
 1. Fork the repository you want to work on.
 2. Set up two remotes.
```
Comment (m-Schlitzer) [inline USING_GIT.md]: You addressed that its added automatically so this should not read "two remotes"

**Comment by m-Schlitzer** [comment]
lgtm

**Pull request merged**